scarlet/observation.py

```diff
@@ -64,7 +63,7 @@ def __init__(
         assert (
                 self.weights.shape == self.images.shape
         ), "Weights needs to have same shape as images"
-
+        self.model_images = np.zeros(self.frame.shape)
```

Comment:
I don't see where this is used. Did I miss something?

Reply:
Good catch. A remainder from a previous attempt.